### DISCUSSION TITLE
`[wrap-git]` vs git submodule

### DISCUSSION THREAD

**Discussion by lucascolley** [mesonbuild/meson#13683] [Q&A] [answered]
In which scenarios would a `[wrap-git]` file under `subprojects/` be preferable to a git submodule in the same location, and vice versa?

**Comment by lucascolley** [comment]
I found https://github.com/scipy/scipy/pull/19255#issuecomment-1745664719 which has effectively already answered this question. Perhaps there is more to say.

**Reply by dcbaker** [reply, marked as answer]
The only thing I'd add to that answer is whether you expect users to be able to find your dependencies on the system or not. If you do expect `pkg-config` to find them, then using a `git-wrap` avoids people needing to initialize submodules they wont use. Otherwise I think Eli summed it up.
